packages/cli/package.json

```diff
@@ -122,8 +121,10 @@
     "truffle-config": "1.1.16",
     "underscore": "^1.9.1",
     "uuid": "^3.3.3",
-    "web3": "1.2.1",
-    "web3-utils": "1.2.0"
+    "web3": "1.2.2",
+    "web3-eth": "1.2.2",
```

Comment:
Are these being added so that they are pinned? They seem to be already pinned on the web3 package itself.

Reply:
This is what yarn failed to properly hoist. On a clean install, I ended up with web3 1.2.2, web3-eth 1.2.2, but web3-eth-contract 1.2.1. We have a transitive dependency to web3 1.2.1, which is installed within the package that requires it, but that's not the case for web3-eth-contract.

Reply:
Ah, I see. It'd be great if we were able to document this here. Shame Crockford decided to disallow comments